internal/worker/rundynamic.go

```diff
@@ -35,10 +34,10 @@ error: Any error that occurred in the runtime/sandbox infrastructure.
 This does not include errors caused by the package under analysis.
 */
 
-func RunDynamicAnalysis(pkg *pkgecosystem.Pkg, sbOpts []sandbox.Option) (result.DynamicAnalysisResults, api.RunPhase, analysis.Status, error) {
-	results := result.DynamicAnalysisResults{
-		StraceSummary: make(result.DynamicAnalysisStraceSummary),
-		FileWrites:    make(result.DynamicAnalysisFileWrites),
+func RunDynamicAnalysis(pkg *pkgmanager.Pkg, sbOpts []sandbox.Option) (analysisrun.DynamicAnalysisResults, analysisrun.DynamicPhase, analysis.Status, error) {
+	results := analysisrun.DynamicAnalysisResults{
+		StraceSummary: make(analysisrun.DynamicAnalysisStraceSummary),
+		FileWrites:    make(analysisrun.DynamicAnalysisFileWrites),
```

Comment:
This code is an example of where I think some effort to shorten the names would be worthwhile.